README.md

```diff
@@ -374,19 +311,30 @@ func main() {
 	}
 
 	// Pass the Filter struct to the FilteredInstanceTypes function of your
-	// selector instance to get a list of filtered instance types and their details
+	// selector instance to get a list of filtered instance types and their details.
 	instanceTypesSlice, err := instanceSelector.FilterInstanceTypes(filters)
 	if err != nil {
 		fmt.Printf("Oh no, there was an error getting instance types: %v", err)
 		return
 	}
 
-	// Pass in your list of instance type details to the appropriate output function
-	// in order to format the instance types as printable strings.
-	maxResults := 100
+	// Pass in the list of instance type details to the SortInstanceTypes function
+	// if you wish to sort the instances based on set filters.
+	sortFilter := "instance-type-name"
+	sortDirection := "ascending"
+	instanceTypesSlice, err = instanceSelector.SortInstanceTypes(instanceTypesSlice, &sortFilter, &sortDirection)
+	if err != nil {
+		fmt.Printf("Oh no, there was an error filtering instance types: %v", err)
+		return
+	}
+
+	// Truncate results and format them for output with your desired formatting function.
+	// All formatting functions can be found here:
+	// https://github.com/aws/amazon-ec2-instance-selector/blob/main/pkg/selector/outputs/outputs.go
```

Comment:
> think a better place to document the different options available would be the [examples](https://github.com/aws/amazon-ec2-instance-selector#examples) section of the README, with some examples of the output as well.

Pointing to the examples section of the readme rather than the outputs.go file does seem better since it also shows an example of output. I just felt that having the library users see the function signatures would be helpful and having access to the implementations would allow them to create their own output function if desired. Perhaps I could point to both the readme and `outputs.go`?

> Related, but I see that most of these options overlap with the CLI options already captured in the README, except for VerboseInstanceTypeOutput(), which seems to only be used in tests. What was this function used for, and is this still needed?

VerboseInstanceTypeOutput() is also called in Main whenever the `--verbose` or `-v` flag is passed in order to get a full printout of instance type specs. So I believe it is still needed. I could however add in an example of output to the readme.

Reply:
On second thought, the output of `--verbose` is incredibly long and I believe it would clutter up the readme. I am thinking it would be best to not include an example of the verbose output.